src/components/grid-aware/BlockQuoteBlock/BlockQuoteBlock.js

```diff
@@ -0,0 +1,28 @@
+import PropTypes from "prop-types";
+import React from "react";
+import s from "./BlockQuoteBlock.module.css";
+
+const BlockQuoteBlock = ({ quote, author }) => {
+  const GridAreaMiddle = () => (
```

Comment:
This subcomponent is probably unnecessary to have in such a simple component, so I would just inline this directly into the main body below.

Reply:
👍 Deleted.